protocol/consistencychecks_test.go

```diff
@@ -3,29 +3,28 @@ package protocol
 import "testing"
 
 var (
-	key = []byte("key")
+	alice = "alice"
+	bob   = "bob"
+	key   = []byte("key")
 )
 
-func doRequestAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
-	requestType int, name string) error {
-	switch requestType {
-	case RegistrationType:
-		request := &RegistrationRequest{
-			Username: name,
-			Key:      key,
-		}
-		res, _ := d.Register(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case KeyLookupType:
-		request := &KeyLookupRequest{
-			Username: name,
-		}
-		res, _ := d.KeyLookup(request)
-		return cc.HandleResponse(requestType, res, name, key)
-	case MonitoringType:
-	case KeyLookupInEpochType:
+func registerAndVerify(d *ConiksDirectory, cc *ConsistencyChecks,
```

Comment:
> Sorry for wasting your time on this :(

You certainly did not waste my time.  I'm happy that you're challenging my position; please continue to do so.   That's how we improve.